src/implementations/vanilla/components/checkbox/_checkbox.scss

```diff
@@ -29,6 +29,10 @@ $_label-color: null !default;
   position: relative;
 }
 
+.ecl-checkbox ~ .ecl-checkbox {
```

Comment:
this causes extra spacing in the multi select component:
![image](https://github.com/ec-europa/europa-component-library/assets/25579221/31c5f0ee-3110-4fc0-9c77-afd305facc3b)